### PR TITLE
update usage -- readme

### PR DESCRIPTION
**Pull request by wyli**
### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ All source code of models (bundles) are tracked in `models/`, and for each disti
...
+```bash
+pip install "monai[fire]"
+python -m monai.bundle download wholeBody_ct_segmentation --bundle_dir "bundles/"
+```
+
+- The commands will download `wholeBody_ct_segmentation` to the current directory's `bundles/` subdirectory.
+- For more downloading options, please run `python -m monai.bundle download -h`
+- For the specific usage of a bundle, please refer to its `docs` folder, for example, `bundles/wholeBody_ct_segmentation/docs`.
+
+To browse the available models, please see [the model browser](https://monai.io/model-zoo.html).
```
Comment (Nic-Ma) [inline README.md]: I would suggest to move this line to the beginning of this section, users usually browse the bundles then download.
And BTW, suggest: "available models" -> "available bundles".

Thanks.

Comment (wyli) [reply README.md]: from that page, I couldn't find any basic commands for how to set things up.

Comment (wyli) [reply README.md]: And BTW, suggest: "available models" -> "available bundles".
> 

given this repo is a 'model' zoo, I think available 'models' make sense.

Comment (Nic-Ma) [reply README.md]: > from that page, I couldn't find any basic commands for how to set things up.

Yeah, @zephyrie @yiheng-wang-nv I think we may consider add the description of how to download in https://monai.io/model-zoo.html?
Then let's put this line "To browse the available models..." to the front.
@wyli What do you think?

Thanks in advance.

Comment (yiheng-wang-nv) [reply README.md]: Maybe we can refer to Hugging Face's `How to clone` button (see also [here](https://huggingface.co/MONAI/example_spleen_segmentation)), and can insert the code @wyli added automatically.

Screen Shot 2023-02-27 at 2 20 05 PM

Comment (yiheng-wang-nv) [reply README.md]: Hi @Nic-Ma , I just create a ticket in:
https://github.com/Project-MONAI/project-monai.github.io/issues/46
As for the potential change in model zoo webpage, maybe we can discuss there, and this PR can be merged first as it does enhances the readme. What do you think?

Comment (Nic-Ma) [reply README.md]: Hi @wyli ,

As @yiheng-wang-nv already plan to enhance the web page, can we move the "browse the available models" line to the beginning?

Thanks.

**Review by yiheng-wang-nv — APPROVED** [review]

**Pull request merged**